docs/pages/api-docs/text-field.md

```diff
@@ -81,8 +81,8 @@ The `MuiTextField` name can be used for providing [default props](/customization
 | <span class="prop-name">onChange</span> | <span class="prop-type">func</span> |  | Callback fired when the value is changed.<br><br>**Signature:**<br>`function(event: object) => void`<br>*event:* The event source of the callback. You can pull out the new value by accessing `event.target.value` (string). |
 | <span class="prop-name">placeholder</span> | <span class="prop-type">string</span> |  | The short hint displayed in the input before the user enters a value. |
 | <span class="prop-name">required</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the label is displayed as required and the `input` element` will be required. |
-| <span class="prop-name">rows</span> | <span class="prop-type">number<br>&#124;&nbsp;string</span> |  | Number of rows to display when multiline option is set to true. |
-| <span class="prop-name">rowsMax</span> | <span class="prop-type">number<br>&#124;&nbsp;string</span> |  | Maximum number of rows to display. |
+| ~~<span class="prop-name">rows</span>~~ | <span class="prop-type">oneOfType([PropTypes.number</span> |  | *Deprecated*. Use `minRows` instead<br><br>Number of rows to display when multiline option is set to true. |
```

Comment:
Oh, so it does not work on `master`. We need to fix this first.

Reply:
Yes, I've created a fix - https://github.com/mnajdova/material-ui/pull/25 Not sure if it is the best, but I am writing the whole PropTypes definition there.

Reply:
Created https://github.com/mui-org/material-ui/pull/27576 for fixing it

Reply:
#27576 was merged, I think we can safely merge this one too now.